tdrs-backend/tdpservice/conftest.py

```diff
@@ -45,7 +45,7 @@ def stt_user():
 
 @pytest.fixture
 def ofa_admin_stt_user():
-    """Return an admin user without an STT for Data Report tests."""
+    """Return an admin user without an STT for Data DataFile tests."""
```

Comment:
If we run it on a deployed website, it will change the database and the app won't work until this gets merged

Reply:
Have you tested a rollback from these migrations? I'm not sure how it will play with the table renaming but Django does support the ability to undo a migration.

Reply:
I was able to test it by just restarting the web container. follow the instructions except after you run the migration, run `docker-compose restart web`